code/modules/mob/living/living.dm

```diff
@@ -1064,7 +1064,7 @@
 	var/mob/living/user = usr
 	if(!istype(over) || !istype(user))
 		return
-	if(!over.Adjacent(src) || !over.canUseTopic(src) || !canUseTopic(over))
+	if(!over.Adjacent(src) || !over.canUseTopic(src) || !user.canUseTopic(over))
```

Comment:
does this work?

why are your checking canUseTopic on the target?

Reply:
canUseTopic checks for incapacitated and stuff right? that's mainly why it's there, to make sure they can actually use the things.

Reply:
Yes but why would you be checking if the mob that's going to be picked up is unconscious instead of the person trying to pick them up?

Reply:
.. Oh.